packages/desktop-client/src/components/mobile/budget/CategoryTransactions.tsx

```diff
@@ -72,10 +71,6 @@ export function CategoryTransactions({
         ) {
           reloadTransactions();
         }
-
-        if (tables.includes('payees') || tables.includes('payee_mapping')) {
```

Comment:
No need for this since this is already handles by the global sync-event listener